handler_test.go

```diff
@@ -207,6 +207,43 @@ func TestPathConfigSetFind(t *testing.T) {
 			want:    "/portmidi",
 			subpath: "foo",
 		},
+		{
+			paths:   []string{"/example/helloworld", "/", "/y", "/foo"},
+			query:   "/x",
+			want:    "/",
+			subpath: "",
```

Comment:
subpath should be `"x"` (similarly for following test cases)

Reply:
Oh great, thanks for the reminder, I am updating the algo.